lint.sh

```diff
@@ -17,6 +17,11 @@
 function lintAllFiles () {
   echo "Running linter on module $1"
   pylint --disable=$2 $1
+  rc=$?
```

Comment:
If all you want to do is exit if the linter fails then you should just set that mode at the top of the script, e.g. with

```bash
set -o errexit
```

That will get rid of both of these extra blocks.

You might also consider adding `set -o nounset` as well, which will help catch typos in variable names (that would otherwise expand to empty strings).